packages/cfd/src/Containers/jurisdiction-modal/__test__/jurisdiction-title-indicator.spec.tsx

```diff
@@ -0,0 +1,194 @@
+import React from 'react';
+import { screen, render } from '@testing-library/react';
+import JurisdictionTitleIndicator from '../jurisdiction-title-indicator';
+import { Jurisdiction } from '@deriv/shared';
+
+describe('JurisdictionTitleIndicator', () => {
+    type TMockProps = {
+        title_indicators: {
+            type: 'displayText' | 'displayIcons';
+            display_text?: string;
+        };
+        type_of_card: 'svg' | 'bvi' | 'vanuatu' | 'labuan' | 'maltainvest';
+        account_status: {
+            authentication: {
+                document: {
+                    status: 'none' | 'pending' | 'verified' | 'expired' | 'rejected' | undefined;
+                };
+                identity: {
+                    services: {
+                        idv: {
+                            status: 'none' | 'pending' | 'verified' | 'expired' | 'rejected' | undefined;
+                        };
+                        onfido: {
+                            status: 'none' | 'pending' | 'verified' | 'expired' | 'rejected' | undefined;
+                        };
+                        manual: {
+                            status: 'none' | 'pending' | 'verified' | 'expired' | 'rejected' | undefined;
+                        };
+                    };
+                };
+                needs_verification: string[];
+            };
+            currency_config: {
+                [k: string]: {
+                    is_deposit_suspended?: 0 | 1;
+                    is_withdrawal_suspended?: 0 | 1;
+                };
+            };
+            p2p_status: 'none';
+            prompt_client_to_authenticate: 0;
+            risk_classification: string;
+            status: string[];
+        };
+        verification_docs: ['document_number' | 'selfie' | 'identity_document' | 'name_and_address' | 'not_applicable'];
+    };
+    const mock_props: TMockProps = {
```

Comment:
```suggestion
    const mock_props: TJurisdictionTitleIndicatorProps = {
```
if it doesn't work, please use Partial<TJurisdictionTitleIndicatorProps> then.
TMockProps type declaration can be removed.
Please check the same cases in other test files. 🙂